master/internal/experiment/bulk_action.go

```diff
@@ -41,6 +41,13 @@ type deleteExperimentOKResult struct {
 	State    experimentv1.State
 }
 
+var completedExperimentStates = []string{"CANCELED", "COMPLETED", "ERROR"}
```

Comment:
please use the model pkg states as the 'source of truth' for terminal/running/etc, and convert them to proto if needed. (so remove these)

Reply:
✅